tests/migration/migration.go

```diff
@@ -669,10 +605,11 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 				Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(errorMessageFormat, pid, stdout, stderr, err))
 
 				// wait for both virtqemud to respawn and all connections to re-establish
+				// TODO flaky by design
```

Comment:
Nit: you're not suggesting what **to do** about it :)

Reply:
Maybe something like this?
https://github.com/kubevirt/kubevirt/pull/12838/commits/27848c3403042fb94f82a6baab3112814bdfbc5a#diff-5034f7c82b377dd518dd147554ad7df560b0cc667f1f0aa08e7a632d1436237aR649-R652
WDYT?